pkgs/tools/networking/fort-validator/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, jansson
+, rsync
+, curl
+, libxml2
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fort-validator";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = "NICMx";
+    repo = "FORT-validator";
+    rev = "v${version}";
+    sha256 = "sha256-sldN/xg+agdunTln2q03XEjSwyDtDJucv+9YizP4nw8=";
+  };
+
+  buildInputs = [
+    jansson
+    rsync
```

Comment:
Is this linking against rsync? If it is run it should be moved to native.

Reply:
I've moved this to native.